api/XDS_PROTOCOL.md

```diff
@@ -209,6 +211,20 @@ multiple `DiscoveryRequests` at a version until a new version is ready.
 An implication of the above resource update sequencing is that Envoy does not
 expect a `DiscoveryResponse` for every `DiscoveryRequest` it issues.
 
+### Resource warming
+
+[`Clusters`](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/cluster_manager.html#cluster-warming) 
+and [`Listeners`](https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/lds#config-listeners-lds)
+go through  `warming` before they can serve requests. This process happens both during 
+[`Envoy initialization`](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/init.html#initialization) 
+and when the `Cluster` or `Listener` is updated. Warming of `Cluster` is completed only when a 
+`ClusterLoadAssignment` response is supplied by management server. Similarly, warming of `Listener`
+is completed only when a `RouteConfiguration` is suppled by management server if the listener 
+refers to an RDS configuration. Management server is expected to provide the EDS/RDS updates during
+warming. If management server does not provide EDS/RDS responses, Envoy will not initialize
+itself during initialization phase and the updates sent via CDS/LDS will not take effect till
```

Comment:
Drop "during initialization phase"?

Reply:
I was in two minds when I wrote this - but decided to add this for more clarity. LMK if you strong feel about removing, I can remove.

Reply:
Up to you. I don't feel too strongly.